File: 1Trimestre/Ejercicio4.java
//Este ejercicio, evalúa el día de la semana dependiendo del numero (ente 1 y 7) introducido por el usuario.
package traduccionejercicios;
import java.util.Scanner;
public class Ejercicio_4{

	public static void main(String[] args) {

		int diaSemana;
		
		System.out.println("Introduzca un dia de la semana del 1 al 7");
	
		Scanner scanner = new Scanner(System.in);
		diaSemana = scanner.nextInt();
		
		switch (diaSemana) {
			case 1:
				System.out.println("El dia de la semana es Lunes");
				break;
			case 2:
				System.out.println("El dia de la semana es Martes");
				break;
			case 3:
				System.out.println("El dia de la semana es Miércoles");
				break;
			case 4:
				System.out.println("El dia de la semana es Jueves");
				break;
			case 5:
				System.out.println("El dia de la semana es Viernes");
				break;
			case 6:
				System.out.println("El dia de la semana es Sábado");
				break;
			case 7:
				System.out.println("El dia de la semana es Domingo");
				break;
			default:
				System.out.println("El dato introducido no es válido");
		}
	}
}

File: 1TrimestrePython/Ejercicio4.py
#Este ejercicio, evalúa el día de la semana dependiendo del numero (ente 1 y 7) introducido por el usuario.
dia=int(input("Introduzca un numero del 1 al 7 para evaluar que día de la semana es"))

if dia==1:
    print("El día introducido es lunes")
elif dia==2:
    print("El día introducido es martes")
elif dia==3:
    print("El día introducido es miércoles")
elif dia==4:
    print("El día introducido es jueves")
elif dia==5:
    print("El día introducido es viernes")
elif dia==6:
    print("El día introducido es sábado")
elif dia==7:
    print("El día introducido es domingo")
else:
    print("El dato introducido no es válido")
